src/_forms.scss

```diff
@@ -60,239 +150,306 @@ category: Common
-    background-color: $pasteup-forms-success-background;
-    border-bottom: 1px solid $pasteup-forms-success-border-colour;
-    border-top: 1px solid $pasteup-forms-success-border-colour;
-    color: $pasteup-forms-success-colour;
-    margin-top: $gs-baseline/2;
-    padding: 7px ($gs-baseline/3)*2;
-}
+// Form field
+// ============================================================================
 
-.form-field__error {
-    color: $pasteup-forms-error-colour;
-    margin-top: $gs-baseline/2;
+.form-field {
+    margin-bottom: rem($gs-baseline);
 }
 
-.form-field__note--below {
+.form-field--no-margin {
     margin-bottom: 0;
-    margin-top: $gs-baseline/2;
 }
 
-.form-field__note--left {
-    float: left;
+.form-field--error {
+    .label {
+        color: $pasteup-forms-error-label-colour;
+    }
+
+    .input-text,
+    .input-text:focus,
+    .input-textarea,
+    .input-textarea:focus {
+        border-color: $pasteup-forms-error-input-border-colour;
+    }
 }
 
-.form-field__note--right {
-    float: right;
-    margin-left: $gs-gutter;
+.form-field__error-message {
+    @include fs-data(3);
+    color: $pasteup-forms-error-message-colour;
+    margin-top: rem(2px);
+    margin-bottom: rem($gs-baseline / 2);
 }
 
-.form-fields__inline {
-    ul {
-        margin: 0;
-        padding: 0;
+.form-note {
+    @include fs-textSans(1);
+    color: $pasteup-forms-note-colour;
+    padding-top: rem($gs-baseline / 2);
+    margin-bottom: rem(8px);
+}
 
-        .form-field {
-            display: inline-block;
-        }
+.form-note--left {
+    @include mq(tablet) {
+        float: left;
     }
+}
 
-    .form-fields__heading {
-        @include fs-bodyHeading(3, true);
-        margin: 0 0 $gs-baseline;
-        display: block;
+.form-note--right {
+    @include mq(tablet) {
+        text-align: right;
+        margin-left: rem($gs-gutter);
     }
 }
 
-.form-fields--date-of-birth .form-field {
-    margin: 0 18px 0 0;
+.form-note--underneath {
+    margin-top: rem($gs-baseline / 2);
+}
+
+.form-legal {
+    @include fs-textSans(3);
+    color: $pasteup-forms-note-colour;
 }
 
+// Form labels and inputs
+// ============================================================================
+
 .label {
+    @include fs-bodyCopy(2);
+    color: $pasteup-forms-label-colour;
     cursor: pointer;
     display: block;
-    margin-bottom: 6px;
 }
 
-.text-input,
-.textarea {
-    border: 1px solid $pasteup-forms-fields-border-colour;
-    @include box-shadow(none);
-    @include box-sizing(border-box);
-    color: $pasteup-forms-fields-colour;
+.label--inline {
     display: inline-block;
-    font-family: $pasteup-forms-fields-font-family;
-    padding: 8px 8px 7px;
-    font-size: 16px;
-    line-height: 1.4;
+}
+
+.input-text,
+.input-textarea {
+    color: $pasteup-forms-input-colour;
+    border: 1px solid $pasteup-forms-input-border-colour;
+    @include f-textSans;
+    @include font-size(16px, 22px);
+    padding: rem(8px) rem(8px) rem(7px);
```

Comment:
These are a little magic numberey in the original styles, I wonder if there's a way we could simplify this?